src/test/harnesses/VoteWeigherBaseHarness.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "../../contracts/middleware/VoteWeigherBase.sol";
+
+// wrapper around the StakeRegistry contract that exposes the internal functions for unit testing.
+contract VoteWeigherBaseHarness is VoteWeigherBase {
+    constructor(
+        IStrategyManager _strategyManager,
+        IServiceManager _serviceManager
+    ) VoteWeigherBase(_strategyManager, _serviceManager) {}
+
+    function getMaxWeighingFunctionLength() public pure returns (uint8) {
+        return MAX_WEIGHING_FUNCTION_LENGTH;
+    }
+
+    function getWeightingDivisor() public pure returns (uint256) {
+        return WEIGHTING_DIVISOR;
+    }
```

Comment:
should these constants maybe just be defined as `public` in the first place?
I'm OK with needing a harness for this, but don't think there's really a very significant downside to having the public default getters exist (technically there is a small function dispatching gas cost increase, but IMO it's fine).

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/d4be2b7f8585ad8ef9171fba9171db253b572e0b